src/Analyzers/CSharp/Tests/UseIndexOrRangeOperator/UseRangeOperatorTests.cs

```diff
@@ -479,5 +468,32 @@ void Goo(Span<int> s)
                 FixedCode = fixedSource,
             }.RunAsync();
         }
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsUseRangeOperator)]
+        public async Task TestWritableType()
```

Comment:
I do not think this test is necessary since its exactly matches the updated test up above